var/spack/repos/builtin/packages/py-pytest-pylint/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPytestPylint(PythonPackage):
+    """Run pylint with pytest and have configurable rule types (i.e.
+    Convention, Warn, and Error) fail the build. You can also specify a
+    pylintrc file.
+
+    """
+
+    homepage = "https://pypi.org/project/pytest-pylint/"
+    pypi = "pytest-pylint/pytest-pylint-0.21.0.tar.gz"
+    git = "https://github.com/pytest-dev/pytest"
+
+    license("MIT")
+
+    version("0.21.0", sha256="88764b8e1d5cfa18809248e0ccc2fc05035f08c35f0b0222ddcfea1c3c4e553e")
+
+    # python_requires
+    depends_on("python@3.7:", type=("build", "run"))
+
```

Comment:
Missing build dep on setuptools

Reply:
I added setuptools at the latest version because it looks like setuptools is always updated to the latest version on testing anyway, and this hasn't changed for a long while so I'm assuming it's safe.

From git workflows in pytest-pylint:
```
- name: Install dependencies
        run: |
          python -m pip install --upgrade pip setuptools wheel
          python -m pip install --upgrade tox tox-py
```

Reply:
If no specific version is required, then no specific version should be added

Reply:
Thank you, I'll get there eventually!